RichClipboard.package/RCPluggableListMorph.class/README.md

```diff
@@ -0,0 +1 @@
+An RCPluggableListMorph enables interaction with the RCClippingsList in the RCClippingsWindow.
```

Comment:
Würde noch genauer spezifizieren, was RCPluggableListMorph ist.

~Ist Counterpart zu RCClippingsList aka LazyListMorph + PluggableListMorph.
~Verantwortlich für die Systembrowser-ähnliche Suchfunktion (einfach in Fenster reintippen)